src/components/Button.tsx

```diff
@@ -16,9 +16,9 @@ interface ButtonProps {
 export function Button({
   children,
   onClick,
-  active,
+  active = false,
   className,
-  style,
+  style = {},
```

Comment:
Just remove it — it's fine to have it be `undefined`.

Reply:
done~

Reply:
pls review，thx~